R-package/src/install.libs.R

```diff
@@ -41,71 +111,65 @@ if (!use_precompile) {
   setwd(build_dir)
 
   # Prepare installation steps
-  cmake_cmd <- "cmake "
-  build_cmd <- "make _lightgbm"
+  cmake_args <- NULL
+  build_cmd <- "make"
+  build_args <- "_lightgbm"
   lib_folder <- file.path(source_dir, fsep = "/")
 
   if (use_gpu) {
-    cmake_cmd <- paste0(cmake_cmd, " -DUSE_GPU=ON ")
+    cmake_args <- c(cmake_args, "-DUSE_GPU=ON")
```

Comment:
switching to a separation of `command` and `args` here because `processx::run()` has that interface. It's easier to keep them separate and jam them back into one string for `system()` then to keep building up one string and break it back apart for `processx::run()`, in my opinion.